From dc35ce2ba774a9284360bfd6532acb527a2f73d7 Mon Sep 17 00:00:00 2001
From: Gilles Dartiguelongue <eva@gentoo.org>
Date: Fri, 7 Dec 2018 11:21:25 +0100
Subject: [PATCH 3/4] Fix install failure when not built with OCaml support

Makefile rules in po/ require builder/index-parse.c to be generated to
allow translation utilities to work on it, however builder/ is
completely masked behind OCaml conditional build even if some tools do
not require it and proper guards are in place already.

Closes: https://bugzilla.redhat.com/show_bug.cgi?id=1614502
---
 Makefile.am | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 7eefacbeb..c0a68151f 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -155,7 +155,6 @@ if HAVE_OCAML
 SUBDIRS += \
 	mllib \
 	customize \
-	builder builder/templates \
 	get-kernel \
 	resize \
 	sparsify \
@@ -169,6 +168,9 @@ SUBDIRS += dib
 endif
 endif
 
+# Tools mixed with more OCAML based tools
+SUBDIRS += builder builder/templates
+
 # Perl tools.
 if HAVE_TOOLS
 SUBDIRS += tools
-- 
2.19.2

